dnsmasq-plan/plan.sh

```diff
@@ -0,0 +1,31 @@
+pkg_name=dnsmasq
+pkg_origin=core
+pkg_version=2.80
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_description="Provides network infrastructure for small networks"
+pkg_upstream_url="http://www.thekelleys.org.uk/dnsmasq/doc.html"
+pkg_license=("GPL")
```

Comment:
I think `GPL-2.0-or-later` more closely matches the licensing on Dnsmasq based on their documentation: 

> Dnsmasq is distributed under the GPL, version 2 or version 3 at your discretion

We should also ensure the `COPYING` and `COPYING-v3` files are distributed with the package build.

Reply:
They are indeed included in the package. Tested a deployment:

```
./hab/pkgs/core/xz/5.2.4/20190115013348/share/doc/xz/COPYING.GPLv2
./hab/pkgs/core/xz/5.2.4/20190115013348/share/doc/xz/COPYING
```